### ISSUE TITLE
Properties should not return Select/Where clauses

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#312]
Properties that return `IEnumerable` should not return results of Linq's `Select` or `Where` clauses.
The reason is that the results is lazily evaluated later on (when iterating over the result), thus probably leading to unexpected behavior.

E.g.:
```C#
public IEnumerable<int> MyProperty
{
    get
    {
        return _myField.Where(_ => _ != 42);
    }
}
```
Here the `Where` clause should be reported.
